AmericanExpress/src/main/java/com/braintreepayments/api/AmericanExpressAnalytics.java

```diff
@@ -0,0 +1,21 @@
+package com.braintreepayments.api;
+
+enum AmericanExpressAnalytics {
+
+    // Get Rewards Balance Events
+    REWARDS_BALANCE_STARTED("amex:rewards-balance:started"),
+    REWARDS_BALANCE_FAILED("amex:rewards-balance:failed"),
+    REWARDS_BALANCE_SUCCEEDED("amex:rewards-balance:succeeded");
+
+    private final String event;
+
+    AmericanExpressAnalytics(String event) {
+        this.event = event;
+    }
+
+    String getEvent() {
+        return event;
+    }
+}
+
+
```

Comment:
Super nit: it looks like most of the files don't have spaces at the end, is that our preference?

Reply:
No there should be newlines - fixed 👍